website_canonical_url/README.rst

```diff
@@ -0,0 +1,61 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+=====================
+Website Canonical URL
+=====================
+
+In Website Builder, eCommerce, Blog, etc. Odoo add parameters in URLs, like category,
+page or sorting. This is a bad thing for SEO because it creates DUST (Duplicate URL,
+Same Text) and Duplicate Content. That is to say, multiple URLs that leads to the same
+page search engine's index.
+
+This module doesn't prevent Odoo to generate such URLs, but it helps to reduce the
+search engine's index size by adding canonical URLs for each page :
```

Comment:
How can I test this please?

I go to runbot in Odoo without this module and:
- http://324224-11-0-99d287.runbot9.odoo.com/shop/category/laptops-11
I go to this runbot with this module and:
- http://3324803-449-32d4d7.runbot2.odoo-community.org/shop/category/laptops-11

So how can I test what you say here?
Thank you!

Reply:
Look for a "link rel=canonical" tag in the html source (in the head part).
You're right, the README could be more explanatory